acestep/training_v2/fixed_lora_module.py

```
@@ -0,0 +1,310 @@
+           ``sample_timesteps()`` (logit-normal).
+        4. Interpolate ``x_t = t * x1 + (1 - t) * x0``.
+        5. Forward through decoder, compute flow matching loss.
+    """
+
+    def __init__(
+        self,
+        model: nn.Module,
+        adapter_config: AdapterConfig,
+        training_config: TrainingConfigV2,
+        device: torch.device,
+        dtype: torch.dtype,
```

Ruff (0.15.0)

[warning] Unused method argument: `dtype`

(ARG002)